### PR TITLE
Add support installing packsgest in the Rosa Fresh

### PR DESCRIPTION
**Pull request by yamah-byv**
Updatre file gpoa/util/rpm.py to add support instaling pakages in Rosa Fresh 12 or above.

Yaroslav Belykh

### CODE REVIEW

File: gpoa/util/rpm.py [hunk 1]
```diff
@@ -36,9 +45,19 @@ def is_rpm_installed(rpm_name):
 class Package:
 
     def __init__(self, package_name):
-        self.__install_command = ['/usr/bin/apt-get', '-y', 'install']
-        self.__remove_command = ['/usr/bin/apt-get', '-y', 'remove']
-        self.__reinstall_command = ['/usr/bin/apt-get', '-y', 'reinstall']
+        distributive=getDistributiveVendor()
```
Comment (NIR-Ginko) [inline gpoa/util/rpm.py:48]: I suggest to use `distribution` as a correct variable name.

**Comment by mastersin** [comment]
У нас установка, вроде бы, использует PackageKit. В какой степени текущий вариант поддержки реализует необходимый функционал? Кроме того, хотелось бы поправки на тип дистрибутива сделать более общим архитектурным механизмом.

**Comment by NIR-Ginko** [comment]
Please squash commits into a single one to avoid meaningless comments.

**Comment by NIR-Ginko** [comment]
Have you tested the code on ROSA Fresh?

**Comment by NIR-Ginko** [comment]
> У нас установка, вроде бы, использует PackageKit. В какой степени текущий вариант поддержки реализует необходимый функционал? Кроме того, хотелось бы поправки на тип дистрибутива сделать более общим архитектурным механизмом.

I suggest to make the code generalization on the next step.

**Review by NIR-Ginko — CHANGES_REQUESTED** [review]

**Comment by mastersin** [comment]
> Please squash commits into a single one to avoid meaningless comments.

I think about it too. But it is not urgent. This can be done in the last step.

File: gpoa/util/rpm.py (hunk 1 above)
Comment (mastersin) [reply gpoa/util/rpm.py:48]: > I suggest to use `distribution` as a correct variable name.

That's what I thought too when I watched the patches.

**Comment by yamah-byv** [comment]
> @yamah-byv, have you tested the code on ROSA Fresh?
Initially, I tested on the original package from Sisyphus.
A package with this patch for Rosa Fresh was compiled yesterday. I will be able to provide full test results on a machine with clear install on April 24.

**Comment by yamah-byv** [comment]
Yes, on pure Rosa Fresh without a bunch of packages from Alt (adp, ssd, python3-module-config, samba-winbind) and manually executing the system-auth script, gpoa cannot read information about the domain.
I withdraw the pull request until better times.

**Comment by NIR-Ginko** [comment]
@yamah-byv , `adp` is an option for `gpupdate` and does not needed in order to function. `sssd` is an option too, `winbind` may be sufficient. Could you please post the error logs?

**Pull request closed without merging**